source/MainMenuState.hx

```diff
@@ -171,6 +175,12 @@ class MainMenuState extends MusicBeatState
 			FlxG.sound.music.volume += 0.5 * FlxG.elapsed;
 		}
 
+		if (CoolUtil.isRecording())
+		{
+			trace('he's recording da game bro"');
```

Comment:
Invalid string declaration, should be self-explanatory.

Reply:
this piece of code is also gonna make a command prompt window pop up *every* time the user goes to the main menu state